### ISSUE TITLE
make-model-moving-on-the-surface

### ISSUE THREAD

**Issue by yanzixiang** [theArcticOcean/CLib#2]
https://www.weiy.city/2020/07/make-model-moving-on-the-surface/

this example seems NOT compatible with VTK-9.1.0.rc2，
Can you have a look at this ?

**Comment by theArcticOcean** [comment]
Yes, I built it based on vtk-8.2.0. What's the issue when you build the project with VTK-9.1.0.rc2 ?
